### ISSUE TITLE
Scrollbars need to be styled

### ISSUE THREAD

**Issue by magdakok** [SaraVieira/make-frontend-shit-again#59]
Scrolling the website is not fun. Could you please colour both horizontal and vertical scrollbar?

I hope it helps:
https://www.webdevelopersnotes.com/tips/html/scrollbar_magnified.gif

**Comment by xuedi** [comment]
> Scrolling the website is not fun. Could you please colour both horizontal and vertical scrollbar?
> 
> I hope it helps: https://www.webdevelopersnotes.com/tips/html/scrollbar_magnified.gif

I agree, the given colors by [magdakok](https://github.com/magdakok) are a must! :-)
